<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OxygenExceptionListWidget</class>
 <widget class="QWidget" name="OxygenExceptionListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="KPushButton" name="moveUpButton">
     <property name="text">
      <string>Move Up</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KPushButton" name="moveDownButton">
     <property name="text">
      <string>Move Down</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KPushButton" name="addButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KPushButton" name="removeButton">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="KPushButton" name="editButton">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" rowspan="6">
    <widget class="QTreeView" name="exceptionListView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
